### DISCUSSION TITLE
Help please chatroomrecorder stops n starts chopping stream into sections - only happening on some streamers

### DISCUSSION THREAD

**Discussion by ssdd99** [SagalVanderklok/ChatRoomRecorder#31] [Q&A] [answered]
Hi
When using chatroomrecorder(1.5.2) on about 10% of streamers the recording will stop n start several times during the stream. some stream is lost due to interruptions in recording. This only happens on a few of the streamers with 99% others recording perfectly.
Attached is a screen copy of "Wanna_breed" which happens to frequently
I will be updating Ff Mpeg shortly but only guessing at this stage
Thanks
ssdd99
![Screen Shot 07-28-24 at 08 42 AM](https://github.com/user-attachments/assets/85ab5b51-c55f-468f-ba20-9072c8abb9bd)

**Comment by SagalVanderklok** [comment, marked as answer]
Hello! The app records public streams only and when a room status is changed from "online" to any other value recording is stopped. That's the reason in the most cases, but sometimes there're some issues with the source stream itself. For the latter case I'll add ffmpeg buffer parameter in the next release

**Reply by ssdd99** [reply]
thanks heaps
